src/WorkBC.Data/Migrations/20240413000723_InsertNocCodes2021.cs

```diff
@@ -14,36 +14,41 @@ namespace WorkBC.Data.Migrations
     public partial class InsertNocCodes2021 : Migration
     {
         private readonly HttpClient _httpClient;
+        private readonly string _ssotBaseUrl;
         private readonly string _ssotUrl;
 
+
         public InsertNocCodes2021() : base()
         {
             _httpClient = new HttpClient();
-            _ssotUrl = Environment.GetEnvironmentVariable("SSOT_URL");
+            _ssotBaseUrl = Environment.GetEnvironmentVariable("SSOT_URL");
+            _ssotUrl = _ssotBaseUrl + "/nocs_nocs";
+           
             if (_ssotUrl == null)
```

Comment:
Please change the check for `_ssotUrl` to `_ssotBaseUrl` and move before setting `_ssotBaseUrl`.

Reply:
Made these changes in the latest commit.